src/providers/twitch/IrcMessageHandler.cpp

```diff
@@ -706,28 +701,6 @@ void IrcMessageHandler::handleUserNoticeMessage(Communi::IrcMessage *message,
     }
 }
 
-void IrcMessageHandler::handleModeMessage(Communi::IrcMessage *message)
-{
-    auto app = getApp();
-
-    auto channel = app->twitch.server->getChannelOrEmpty(
-        message->parameter(0).remove(0, 1));
-
-    if (channel->isEmpty())
-    {
-        return;
-    }
-
-    if (message->parameter(1) == "+o")
-    {
-        channel->modList.append(message->parameter(2));
-    }
-    else if (message->parameter(1) == "-o")
-    {
-        channel->modList.append(message->parameter(2));
-    }
-}
-
```

Comment:
Does TMI not notify of +/-o anymore? Feels weird to take this out.

Reply:
TMI does not emit `MODE` message at all.

Reply:
Regardless, `channel->modList` was completely unused either way.

Reply:
It used to emit `MODE`s, that **was even documented**. :rocket:

Reply:
Given @mm2pl's comment, I assume it's fine to completely remove this. Resolving this conversation.